src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6314,28 +6315,33 @@ namespace SimAirServingZones {
 				CalcSysSizing( AirLoopNum ).InpDesCoolAirFlow = CalcSysSizing( AirLoopNum ).DesCoolVolFlow * FractionOfAutosize;
 				FinalSysSizing( AirLoopNum ).InpDesCoolAirFlow = FinalSysSizing( AirLoopNum ).DesCoolVolFlow * FractionOfAutosize;
 			} else if ( SELECT_CASE_var == FlowPerCoolingCapacity ) {
-				if ( FinalSysSizing( AirLoopNum ).CoolingCapMethod == FractionOfAutosizedCoolingCapacity ) {
+				if ( FinalSysSizing( AirLoopNum ).CoolingCapMethod == FractionOfAutosizedCoolingCapacity || FinalSysSizing( AirLoopNum ).ScaledCoolingCapacity == DataSizing::AutoSize ) {
```

Comment:
Get input already checks the value of ".ScaledHeatingCapacity" and for this capacity sizing method it will not be set to autosize. I will make this change.